www/templates/index.php

```diff
@@ -1,63 +1,78 @@
+                    </span>
+                </div>
+                <?php else: ?>
+                <div class="dcf-form-group">
+                    <label id="theAliasLabel" for="theAlias">Alias</label>
+                    <span>
+                        <input id="theAlias" name="theAlias" type="text" aria-labelledby="theAliasLabel" aria-describedby="theAliasDesc" disabled>
+                        <span class="dcf-form-help" id="theAliasDesc" tabindex="-1">If you would like to control the <abbr class="dcf-txt-sm" title="Uniform Resource Locator">URL</abbr>, then enter the alias you would like to use. Please <a href="./?login">log in</a> to use this feature.</span>
+                    </span>
+                </div>
+                <?php endif; ?>
             </fieldset>
             <fieldset>
-              <legend class="dcf-legend dcf-pt-4 dcf-txt-lg">Google Analytics Campaign Tagging</legend>
-              <p class=" dcf-txt-sm">Add your campaign information here and it will be automatically added to your URL when redirected.</p>
-              <ol>
-                <li>
-                  <label class="dcf-label" for="gaName" class="element">Campaign Name<span class="required">*</span> <span class="helper">product, promo code, or slogan</span></label>
-                  <input class="dcf-input-text" name="gaName" id="gaName" type="text" />
-                </li>
-                <li>
-                  <label class="dcf-label" for="gaMedium">Medium<span class="required">*</span> <span class="helper">marketing medium: email, web, banner</span></label>
-                  <input class="dcf-input-text" name="gaMedium" id="gaMedium" type="text" />
-                </li>
-                <li>
-                  <label class="dcf-label" for="gaSource">Source<span class="required">*</span> <span class="helper">referrer: google, facebook, twitter</span></label>
-                  <input class="dcf-input-text" name="gaSource" id="gaSource" type="text" />
-                </li>
-                <li>
-                  <label class="dcf-label" for="gaTerm">Term <span class="helper">identify the keywords</span></label>
-                  <input class="dcf-input-text" name="gaTerm" id="gaTerm" type="text" />
-                </li>
-                <li>
-                  <label class="dcf-label" for="gaContent">Content <span class="helper">use to differentiate ads (A/B testing)</span></label>
-                  <input class="dcf-input-text" name="gaContent" id="gaContent" type="text" />
-                </li>
-              </ol>
+                <legend class="dcf-bold dcf-txt-lg">Google Analytics Campaign Tagging</legend>
+                <p class="dcf-txt-sm">Add your campaign information here and it will be automatically added to your URL when redirected.</p>
+                <div class="dcf-form-group">
+                    <label id="gaNameLabel" for="gaName">Campaign Name <small class="dcf-required">Required</small></label>
```

Comment:
The google analytics inputs need to be conditionally required.  The user should be able to add url with google analytics settings but if want them they need to enter all the required fields.

Reply:
I added the `required` attributes based on the 'required' indicators (asterisks) already present in the labels. Let's chat about this tomorrow.

Reply:
I temporarily removed the `required` attributes and made an issue to conditionally add them instead: #29.